.s2i/bin/run

```diff
@@ -1,11 +1,27 @@
 #!/bin/bash
-
 set -eo pipefail
 
 # Copy over the required notebooks and data
 echo "copying notebooks"
-dir_name=notebooks-$(date +"%F-%H-%M")
+dir_name=$GIT_REPO_NAME-$(date +"%F-%H-%M")
 mkdir /opt/app-root/src/$dir_name
-cp -Rf /opt/app-root/backup/notebooks/* /opt/app-root/src/$dir_name
+notebooks_location=/opt/app-root/src/$dir_name/
+cp -Rf /opt/app-root/backup/notebooks/* $notebooks_location
+
+# Generate README.md
+echo "generating README.md"
+readme_location=$notebooks_location/README.md
+touch $readme_location
+echo "# $GIT_REPO_NAME" > $readme_location
+echo "#### Version: $GIT_TAG" >> $readme_location
+echo "#### Author: $GIT_TAG_AUTHOR" >> $readme_location
+echo "" >> $readme_location
+echo "$GIT_REPO_DESC <br>" >> $readme_location
+echo "" >> $readme_location
+echo "### Source: <br>" >> $readme_location
+echo "This notebook image has been created from notebooks available at: " >> $readme_location
+echo "[$GIT_TAG_COMMIT_SHA](${GIT_REPO_URL:0:-4}/tree/$GIT_TAG_COMMIT_SHA/notebooks)" >> $readme_location
```

Comment:
```suggestion
echo "generating README.md"
cat <<EOF > $notebooks_location/README.md
# "$GIT_REPO_NAME"

#### Version: "$GIT_TAG"
#### Author: "$GIT_TAG_AUTHOR"

"$GIT_REPO_DESC"<br>

### Source: <br>
This notebook image has been created from notebooks available at:
[$GIT_TAG_COMMIT_SHA](${GIT_REPO_URL:0:-4}/tree/$GIT_TAG_COMMIT_SHA/notebooks)
EOF
```
Newline can be easily incorporated in above line, with additional command execution.
I dont have major experience with shell script. WDYT?

Reply:
Yeah that looks a lot better :+1: